### ISSUE TITLE
Using the model with highest validation accuracy or lowest validation loss? 

### ISSUE THREAD

**Issue by guneytombak** [Sebi1128/DeepLearningProject#9]
We should save the model parameters with one of those and use on the test that one.

**Comment by guneytombak** [comment]
It is obvious that the highest validation accuracy is much better. Solved and compared with commit a7489b6efb313c711a27cd5a35a06ad4c11ab52d
